db/migrate/20150129120103_add_default_to_category_ordinal.rb

```diff
@@ -0,0 +1,5 @@
+class AddDefaultToCategoryOrdinal < ActiveRecord::Migration
+  def change
+    change_column :comfy_cms_categories, :ordinal, :integer, default: 999
```

Comment:
Ahh, I read it as 'you should be thrown in jail' :) No worries will leave as is.